flake8_plugins/assert_expected_actual.py

```diff
@@ -0,0 +1,113 @@
+import ast
+import json
+from pathlib import Path
+import re
+
+
+MSG = 'PTA100 expected should come before actual'
+
+
+def is_literal(node):
+    return (
+        isinstance(node, ast.Constant) or
+        (
+            (
+                isinstance(node, ast.List) or
+                isinstance(node, ast.Set) or
+                isinstance(node, ast.Tuple)
+            ) and
+            all(is_literal(elt) for elt in node.elts)
+        ) or
+        (
+            isinstance(node, ast.Dict) and
+            all(is_literal(key) for key in node.keys) and
+            all(is_literal(value) for value in node.values)
+        )
+    )
+
+
+def classify_arg(node):
+    info = {'type': type(node).__name__, 'literal': is_literal(node)}
+    if info['type'] == 'Name':
+        for keyword in ['actual', 'expected']:
+            if re.search(keyword, node.id, re.IGNORECASE):
+                info[keyword] = True
+    return info
+
+
+def classify_call(node):
+    info = {'args': len(node.args)}
+    if info['args'] == 2:
+        left, right = map(classify_arg, node.args)
+        info['left'] = left
+        info['right'] = right
+        if (
+            left.get('expected') or
```

Comment:
nit: instead of chaining `or` statements here you can just use the [`any`](https://docs.python.org/3/library/functions.html#any) function from the python stdlib.